FrontEnd/src/Pages/PlantProfile.js

```diff
@@ -4,48 +4,51 @@ import { Box, Typography, Card, CardContent, Grid, LinearProgress, Button, Circu
 import { styled } from '@mui/system';
 import Calendar from 'react-calendar'; 
 import 'react-calendar/dist/Calendar.css'; 
-
-
-const mockPlantData = {
-  id: 1,
-  name: 'Angelique Pink Tulip',
-  location: 'Front Yard',
-  needsWater: true,
-  imageUrl: 'https://cdn11.bigcommerce.com/s-i7i23daso6/images/stencil/1280x1280/products/12276/18429/Tulip_Angelique_0002670_2__75432.1629189655.jpg?c=1', 
-  sensors: {
-    moisture: 75, // Example moisture level percentage
-    temperature: 22, // Example temperature in Celsius
-    light: 60, // Example light level percentage
-  },
-  statistics: {
-    daysSinceWatered: 3,
-    averageGrowthRate: '1.2 cm/day',
-  },
-  wateringFrequency: 'Daily', 
-  preferredSunlight: 'Full Sun',
-  soilType: 'Loamy',
-  careTips: 'Water daily in the morning, ensure full sunlight exposure.',
-};
+import axios from 'axios';
+import 'D:/GardianWateringSystem/LastVersion/FrontEnd/src/App.css';

```

Comment:
Is it possible to add a relative path from the root of the Frontend project? Other developer might not have the D drive in their machine.